transformer.go

```diff
@@ -305,16 +307,38 @@ func (tf *transformer) transformAsm(args []string) ([]string, error) {
 
 	flags = alterTrimpath(flags)
 
-	// The assembler runs twice; the first with -gensymabis,
-	// where we continue below and we obfuscate all the source.
-	// The second time, without -gensymabis, we reconstruct the paths to the
-	// obfuscated source files and reuse them to avoid work.
+	// Add the original source directory to the include path so that
+	// local header files like "abi_amd64.h" can still be found.
+	if len(paths) > 0 {
+		flags = append(flags, "-I", filepath.Dir(paths[0]))
```

Comment:
this feels dangerous; we want to limit the use of original non-obfuscated source files as much as possible. so I'd avoid doing this if possible. if we need files like `abi_amd64.h` in unobfuscated form, then we should explicitly carry those over.